internal/machine/openrc/unit.go

```diff
@@ -65,9 +65,11 @@ func (s ServiceUnit) OverrideCmd(cmd string) error {
 		return fmt.Errorf("no k3s binary found (?)")
 	}
 	cmd = strings.ReplaceAll(cmd, k3sbin+" ", "")
-	svcDir := filepath.Join(s.rootdir, fmt.Sprintf("/etc/rancher/k3s/%s.env", s.name))
+	envFile := filepath.Join(s.rootdir, fmt.Sprintf("/etc/rancher/k3s/%s.env", s.name))
+	env := make(map[string]string)
+	env["command_args"] = fmt.Sprintf(`"%s >>/var/log/%s.log 2>&1"`, cmd, s.name)
```

Comment:
```suggestion
	env["command_args"] = fmt.Sprintf("%s >>/var/log/%s.log 2>&1", cmd, s.name)
```

Reply:
seems that was it! https://github.com/c3os-io/c3os/runs/7509171794?check_suite_focus=true